controllers/metallb_controller.go

```diff
@@ -67,7 +70,13 @@ var ManifestPath = "./bindata"
 
 func (r *MetallbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = context.Background()
-	_ = r.Log.WithValues("metallb", req.NamespacedName)
+	logger := r.Log.WithValues("metallb", req.NamespacedName)
+
+	if req.Name != defaultMetallbCrName {
```

Comment:
I am not sure I understand why we have the above check the name is in the CRD definition and if u try to apply random objectname it won't even come to the reconcile loop, I haven't seen this in the operator examples I have seen so just curious what is the problem u trying to solve here ?

Reply:
We want to ensure only one metallb CR is the configuration source. It can be done in a few ways. The simplest one is to only watch a CR with a specific name.